jmclient/jmclient/blockchaininterface.py

```diff
@@ -49,7 +49,7 @@ def query_utxo_set(self, txouts, includeconf=False):
         # address and output script contain the same information btw
 
     @abc.abstractmethod
-    def estimate_fee_per_kb(self, N):
+    def estimate_fee_per_kb(self, N, quiet=False):
```

Comment:
On the general concept of using a "quiet" variable in this PR and future: I'm against it.

The reason is, our logging code should handle this. I remember that some logging libraries have a lower level even than debug, usually called "trace", I think. If our current logging doesn't have this, we should just add it.

Also, if there are cases where you're applying this "quiet" to `log.info` calls, consider replacing them with `log.debug` calls.

Reply:
Fee estimation logs is something that should be `log.info` IMO, it's important information for the user. Problem is that `direct_send()` calls `estimate_fee_per_kb()` twice and actually uses only second result in final tx. This was kinda quick hacky fix for that. Will look again into this and try to solve in a less hacky way.

Reply:
It looks as if this might be the best approach if we did want to add a TRACE logging level: https://stackoverflow.com/a/22586200 ... but I haven't tried it out yet. Would be added in `jmbase/support.py`.

Reply:
I still don't see how introducing new log level will solve the issue. Fee rate should be logged in higher log level, just not in the case when estimation that will not be used at the end is made.

Reply:
Removed "quiet" argument, this can be addressed later, is not directly related to switch from minrelaytxfee to mempoolminfee.